runner/hooks/completed.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# shellcheck source=runner/logger.bash
+source logger.bash
+
+log.notice "Running ARC Completed Hooks"
```

Comment:
```suggestion
log.notice "Running ARC Job Completed Hooks"
```

We'd better say it's for jobs, or it can be read like "Runner Started Hook" which sounds like something else!